specs/fjord/exec-engine.md

```diff
@@ -0,0 +1,97 @@
+# L2 Execution Engine
+
+## Fees
+
+### L1-Cost fees (L1 Fee Vault)
+
+#### Fjord L1-Cost fee changes (FastLZ estimator)
+
+Fjord updates the L1 cost calculation function to use a FastLZ-based compression estimator.
+The L1 cost is computed as:
+
+```
+l1FeeScaled = l1BaseFeeScalar*l1BaseFee*16 + l1BlobFeeScalar*l1BlobBaseFee
+l1CostSigned = (intercept + fastlzCoef*fastlzSize + uncompressedTxCoef*uncompressedTxSize) * l1FeeScaled / 1e12
+l1Cost = uint256(max(0, l1CostSigned))
+```
+
+Where:
+
```

Comment:
This could be good to have as a markdown table that defines variables up front. It could include their type and a description

Reply:
For values that are constants, you could define them as constants up front similar to this: https://github.com/ethereum/consensus-specs/blob/dev/ssz/simple-serialize.md#constants

Then a rationale section would be great to explain them